config/owasp/suppressions.xml

```diff
@@ -1,94 +1,94 @@
             Revisiting in a few months when SpringBoot 3 is released
         </notes>
         <cve>CVE-2016-1000027</cve>
     </suppress>
 
-    <suppress until="2022-09-01">
+    <suppress>
         <notes><![CDATA[No new version of Tomcat available with a fix -
          the latest version available at the moment (10.1.0-M16) is compromised]]></notes>
         <cve>CVE-2022-34305</cve>
     </suppress>
 
-    <suppress until="2022-09-01">
+    <suppress>
         <notes><![CDATA[tomcat-embed-core-9.0.64.jar (pkg:maven/org.apache.tomcat.embed/tomcat-embed-core@9.0.64,
         cpe:2.3:a:apache:tomcat:9.0.64]]></notes>
         <cve>CVE-2022-34305</cve>
     </suppress>
 
-    <suppress until="2022-10-10">
+    <suppress>
         <notes><![CDATA[Junk CVE, only affects Amazon Corretto when hot patch is enabled]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.apache\.logging\.log4j/log4j\-api@.*$</packageUrl>
         <cve>CVE-2022-33915</cve>
     </suppress>
 
-    <suppress until="2022-11-11">
+    <suppress>
         <notes><![CDATA[file name: spring-security-crypto-5.7.2.jar]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
         <vulnerabilityName>CVE-2020-5408</vulnerabilityName>
     </suppress>
 
-    <suppress until="2022-09-09">
+    <suppress>
         <notes><![CDATA[transitive dependency in au.com.dius.pact file name: tika-core-1.27.jar]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.apache\.tika/tika\-core@.*$</packageUrl>
         <cve>CVE-2022-25169</cve>
         <cve>CVE-2022-30126</cve>
         <cve>CVE-2022-30973</cve>
     </suppress>
 
-    <suppress until="2022-09-09">
+    <suppress>
         <notes><![CDATA[transitive dependency in au.com.dius.pact file name: xercesImpl-2.12.1.jar]]></notes>
         <packageUrl regex="true">^pkg:maven/xerces/xercesImpl@.*$</packageUrl>
         <cve>CVE-2022-23437</cve>
         <cve>CVE-2017-10355</cve>
     </suppress>
 
-    <suppress until="2022-09-09">
+    <suppress>
         <notes><![CDATA[transitive dependency in au.com.dius.pact file name: xercesImpl-2.12.1.jar]]></notes>
         <packageUrl regex="true">^pkg:maven/com.ibm.icu/icu4j@.*$</packageUrl>
         <cve>CVE-2018-18928</cve>
     </suppress>
 
-    <suppress until="2022-09-15">
+    <suppress>
         <notes><![CDATA[
    file name: spring-cloud-commons-2.1.1.RELEASE.jar
    ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.springframework\.cloud/spring\-cloud\-commons@.*$</packageUrl>
         <cve>CVE-2022-31569</cve>
     </suppress>
-    <suppress until="2022-09-15">
+    <suppress>
         <notes><![CDATA[
    file name: jakarta.annotation-api-1.3.5.jar
    ]]></notes>
         <packageUrl regex="true">^pkg:maven/jakarta\.annotation/jakarta\.annotation\-api@.*$</packageUrl>
         <cve>CVE-2022-31569</cve>
```

Comment:
Can you roll this whole file back to master?

Reply:
yeah will do now